### DISCUSSION TITLE
where is the banner file?

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [hassio-addons/addon-ssh#596] [Q&A] [answered]
I would like to edit the banner that appears when the terminal opens--the one that appears if you execute ```ha banner``` at the prompt. I cannot find it in any of the usual places. Where is the file that contains it?

**Comment by frenck** [comment, marked as answer]
It is baked into the "ha" command and isn't customizable.

**Reply by Cold-Lemonade** [reply]
Thanks. I wanted to colorize the banner. What a bummer!
